pkg/cloud/aws/services/elb/loadbalancer.go

```diff
@@ -259,27 +267,22 @@ func (s *Service) deleteClassicELBAndWait(name string) error {
 		LoadBalancerNames: aws.StringSlice([]string{name}),
 	}
 
-	checkForELBDeletion := func() (done bool, err error) {
+	if err := wait.WaitForWithRetryable(wait.NewBackoff(), func() (done bool, err error) {
 		out, err := s.scope.ELB.DescribeLoadBalancers(input)
+		if err != nil {
+			if code, ok := awserrors.Code(err); ok && code == "LoadBalancerNotFound" {
```

Comment:
Do you want to use a constant for this code?

Reply:
Indeed, I had overlooked updating it. Fixed now.